var/spack/repos/builtin/packages/shengbte/package.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack import *
+
+
+class Shengbte(MakefilePackage):
+    """ShengBTE is a software package for solving the Boltzmann Transport
+    Equation for phonons."""
+
+    homepage = "www.shengbte.org"
+    url      = "www.shengbte.org/downloads/ShengBTE-v1.1.1-8a63749.tar.bz2"
+
+    version('1.1.1-8a63749', sha256='43920740d19ae854c8ecae0b648acfdf1d7726ca4c2b44f1a1684457f2f88522')
+
+    depends_on('mpi')
+    depends_on('spglib')
+    depends_on('intel-mkl')
```

Comment:
```suggestion
    depends_on('mkl')
```
Both `intel-mkl` and `intel-parallel-studio` provide MKL libraries. If it can be any lapack provider, you should change it to `depends_on('lapack')`

Reply:
shengbte requires mkl.